tool.go

```diff
@@ -750,7 +750,17 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		r := syscall.Rlimit{}
+
+		err := syscall.Getrlimit(syscall.RLIMIT_STACK, &r)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "gopherjs: failed to get stack size limit: %v", err)
+			os.Exit(1)
```

Comment:
I don't see evidence that we must treat this error differently than another error in this scope. So, doing `return fmt.Errorf(...)` seems better to me, if it works.

Basically, I see doing this `syscall.Getrlimit` call as just another of the steps of the `runNode` process. If executing `node` fails, we return error. Why should this be different?